reference/docs-conceptual/components/ise/object-model/The-PowerShellTab-Object.md

```diff
@@ -34,16 +37,22 @@ Supported in Windows PowerShell ISE 3.0 and later, and not present in earlier ve
 Runs the given script in the PowerShell tab.
 
 > [!NOTE]
-> This method only works on other PowerShell tabs, not the PowerShell tab from which it is run. The script block is run and any value that is returned from the script is returned to the run environment from which you invoked the command. If the command takes longer to run than the **millesecondsTimeout** value specifies, then the command fails with an exception: "The operation has timed out."
+> This method only works on other PowerShell tabs, not the PowerShell tab from which it is run. The
+> script block is run and any value that is returned from the script is returned to the run
+> environment from which you invoked the command. If the command takes longer to run than the
+> **millesecondsTimeout** value specifies, then the command fails with an exception: "The operation
+> has timed out."
 
 **Script** - System.Management.Automation.ScriptBlock or String
 The script block to run.
 
-**\[useNewScope\]** -  Optional Boolean that defaults to **$true**
-If set to **$true**, then a new scope is created within which to run the command. It does not modify the runtime environment of the PowerShell tab that is specified by the command.
+**\[useNewScope\]** -  Optional Boolean that defaults to `$true`
```

Comment:
Validate parameter casing.

Reply:
Not params in this case Object methods and casing appears correct. if there is something I'm missing I'm open to it:)